example/bosh-lite-manifest-template.yml

```diff
@@ -317,3 +322,13 @@ pruner_properties: ((merge))
 metricscollector_properties: (( merge ))
 eventgenerator_properties: (( merge ))
 scalingengine_properties: (( merge ))
+default_dbs: (( merge ))
```

Comment:
This is getting very sloppy. For now I will allow it.
The property overrides and stubs should just be overriding the job properties rather than creating even more properties.